core/src/main/java/com/digitalpebble/stormcrawler/bolt/SiteMapParserBolt.java

```diff
@@ -373,15 +368,12 @@ public void declareOutputFields(OutputFieldsDeclarer declarer) {
      * Examines the first bytes of the content for a clue of whether this document is a sitemap,
      * based on namespaces. Works for XML and non-compressed documents only.
      */
-    private final boolean sniff(byte[] content) {
+    private boolean sniff(byte[] content) {
```

Comment:
Declaring a private method `final` is redundant because you can not override a private function and are therefore handled like being `final` functions. This is basically like declaring the methods of an public interface as public.